packages/app-cli/gulpfile.js

```diff
@@ -1,11 +1,18 @@
 const gulp = require('gulp');
 const fs = require('fs-extra');
 const utils = require('@joplin/tools/gulp/utils');
+const { execSync } = require('child_process');
 
 const tasks = {};
 
+const buildOneNoteConverter = () => {
+	const profile = process.env.NODE_ENV === 'production' ? '--release' : '--debug';
+	return execSync(`yarn dlx wasm-pack build ../onenote-converter --target nodejs ${profile}`);
+};
+
 tasks.prepareBuild = {
 	fn: async () => {
+		buildOneNoteConverter();
```

Comment:
For now, let's not support CLI. The problem is that the CLI app is installed from source on many different operating systems so we might break this for very little benefit. It's more important to support the desktop app because I expect that's what people will want to use when migrating from OneNote.

It should however be clear that OneNote import is not supported on CLI, either by not listing it in the importers or by displaying a useful error message.